### DISCUSSION TITLE
Names of imported Objects

### DISCUSSION THREAD

**Discussion by itsschmidt** [pimcore/pimcore#17366] [Data-Hub]
When importing objects from a CSV the new DataObjects are named by "imported-{randomnumber}" by DataHub. Where can I map a column from my CSV to get it as the DataObects name? 

thanks!

**Comment by khenow1978** [comment]
Hi.. do you mean something like that:

![image](https://github.com/user-attachments/assets/4b2f1499-d61f-45a7-bedf-14db37d6a370)

Assign the column that contains the unique (important!) key to the SYSTEM key value as Field.. that is supposed to create the name for the object in Pimcore.
